flow_screen_components/richTextInputFSC/force-app/main/default/lwc/inputRichTextFSC/inputRichTextFSC.js

```diff
@@ -15,6 +16,8 @@ export default class inputRichTextFSC_LWC extends LightningElement {
     @api label;
     @api characterLimit;
     @api required = false;
+    @api disabledCategories = '';
```

Comment:
The `<lightning-input-rich-text>` component will throw an indexOf exception, if disabledCategories is not initialized with an empty string.